src/processes/ndvi.js

```diff
@@ -0,0 +1,20 @@
+const Process = require('../processgraph/process');
+
+// TODO: assumes bands have been labelled before
```

Comment:
This is not how the NDVI function is planned to work. It should check the metadata for the common names...

Reply:
Do you mean one has to check the STAC metadata for appropriate band synonyms?

Reply:
Yes, the back-end should easily be able to do that.